scalafix-core/shared/src/main/scala/scalafix/util/TokenClasses.scala

```diff
@@ -1,29 +1,32 @@
 package scalafix.util
 
-import scala.meta.internal.classifiers.classifier
+import scala.meta.classifiers.Classifier
 import scala.meta.tokens.Token
 import scala.meta.tokens.Token._
 
-@classifier
 trait Whitespace
 object Whitespace {
-  def unapply(token: Token): Boolean = {
-    token.is[Space] || token.is[Tab] || token.is[Newline] || token.is[FF]
-  }
+  implicit val WhitespaceClassifier: Classifier[Token, Whitespace] =
+    new Classifier[Token, Whitespace] {
```

Comment:
sigh, this would be so much nicer with a SAM

```scala
implicit val WhitespaceClassifier: Classifier[Token, Whitespace] =
  (token: Token) => token.is[Space] || token.is[Tab] || token.is[Newline] || token.is[FF]
```

but we cross-compile for 2.11, so 👍